File: leetcode-cn/1832.0_Check_if_the_Sentence_Is_Pangram.py
'''
执行用时：36 ms, 在所有 Python3 提交中击败了75.06% 的用户
内存消耗：15.1 MB, 在所有 Python3 提交中击败了7.65% 的用户
通过测试用例：79 / 79
'''
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        return 26 == len(set(sentence))


'''
执行用时：36 ms, 在所有 Python3 提交中击败了75.06% 的用户
内存消耗：14.8 MB, 在所有 Python3 提交中击败了95.31% 的用户
通过测试用例：79 / 79
'''
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        x = 0
        for ch in sentence:
            x |= (1 << (ord(ch) - ord('a')))
        return x == ((1 << 26) - 1)
        

File: csharp/1832.0_Check_if_the_Sentence_Is_Pangram.cs
/*
执行用时：80 ms, 在所有 C# 提交中击败了14.81% 的用户
内存消耗：37.2 MB, 在所有 C# 提交中击败了11.11% 的用户
通过测试用例：79 / 79
*/
public class Solution {
    public bool CheckIfPangram(string sentence) {
        return (new HashSet<char>(sentence)).Count == 26;
    }
}
